blocks/large-manual-promo-block/features/large-manual-promo/default.jsx

```diff
@@ -78,7 +78,9 @@ const LargeManualPromo = ({ customFields }) => {
 					>
 						{headline}
 					</Conditional>
-				) : null}
+				) : (
+					headline
```

Comment:
Hi @malavikakoppula ,
The `Conditional` component will still return the `children` if the condition is falsey. I'd suggest that instead we remove the ternary at Line 71 as it's doing the same thing that the `Conditional` component is doing. I'll put a linmk to my PR up for `medium-manual-promo` in a moment.

Reply:
ah Thanks, removed the ternary operator